modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -178,6 +178,14 @@
 			"jspf"
 		]
 	},
+	{
+		"from": "CookieUtil.get(HttpServletRequest, String, boolean)",
+		"issueKey": "LPS-192011",
+		"newImports": [
+			"com.liferay.portal.kernel.cookies.CookiesManagerUtil"
+		],
+		"to": "CookiesManagerUtil.getCookieValue(param#1#, param#0#, param#2#)"
+	},
```

Comment:
@MicaelleSilvaa you missed one pattern on this check. There should be a case where there are only two parameters. Did you take a look at the test file before deleted? Please add this case where the two parameters changed places.

After that send the new pr to @natocesarrego

Reply:
Close this PR with the link to the new PR.

Reply:
New pr: https://github.com/natocesarrego/liferay-portal/pull/1404